### ISSUE TITLE
Allow all users to access the app whatever their user agent is set to

### ISSUE THREAD

**Issue by dave1010** [benschwarz/currency.io#7]
It seems a shame to not allow users to use the app if their user agent doesn't match /(iPhone|iPod|webOS|Android)/. Could you add a link to the app or at least match the user agent against /Mobile/. Thanks.

**Comment by benschwarz** [comment]
I'm not sure what this would achieve… What devices would this additionally allow?

**Comment by dave1010** [comment]
It would be great if desktop devices could use the app.

As for mobiles, it would allow Opera Mobile and the native Symbian (Nokia) & Blackberry browsers. These are about 50% of the mobile browser market worldwide (http://gs.statcounter.com/#mobile_browser-ww-monthly-201106-201108)

**Comment by avelis** [comment]
I have a feeling that supporting lower end browsers starts to degrade at the UI/UX from this project. It comes to a point where you might as well use a regular website that does all the heavy lighting server side. Now, that being said, you are more than welcome to fork the project and allow it yourself.

**Comment by benschwarz** [comment]
You got it, using currency.io on a desktop makes absolutely no sense what so ever. (Clicking buttons for number input?)

Thankfully Opera mobile has a fantastic simulator, Symbian, however I've never built for, so… I'm not sure. But I'm sure its worth checking out how it displays (You're welcome to fork the project and give us some bug reports to start with).

**Comment by mcepl** [comment]
I would add Firefox OS phone as another candidate which is discriminated against. And of course, it is always bad to sniff User-Agent. Why do you do it at all?

**Comment by benschwarz** [comment]
@mcepl I understand the short comings of browser sniffing. 

Ideally the test would be viewport / touch enabled devices — as I've stated many, many times on this project: 

This project _was_ a proof of concept, we tried to see how nice we could make something for iOS based devices, because thats what we use. As you'll note, the project hasn't been updated in a few years — we have no interest in developing it further than its initial state.

If you're interested in replacing the user-agent detection with something else, than I'd love to see your pull request.
